assembly/src/docs/6_TORNADO_FLAGS.md

```diff
@@ -63,3 +63,9 @@ It enables profiler information such as `COPY_IN`, `COPY_OUT`, compilation time,
 * `-Dtornado.opencl.compiler.options=LIST_OF_OPTIONS`:  
 It allows to pass the compile options specified by the OpenCL ``CLBuildProgram`` [specification](https://www.khronos.org/registry/OpenCL/sdk/1.0/docs/man/xhtml/clBuildProgram.html) to TornadoVM at runtime. By default it doesn't enable any. 
 
+
+##### Optimizations
+
+* `-Dtornado.enable.fma=True`:  
+It enables Fused-Multiply-Add optimizations. This option is enabled bu default. However, for some platforms, such as Xilinx with OpenCL 1.0, this option must be disabled. 
```

Comment:
bu -> by. I am not sure if it is generic to all Xilinx devices. I think it is a problem concerning the runtime so perhaps this is more accurate. "platforms, such as the Xilinx FPGAs using the SDAccel 2018.2, this option must be disabled as it causes runtime errors."

Reply:
Done